lib/src/constants/tiles_constants.dart

```diff
@@ -15,4 +15,7 @@ abstract class TilesConstants {
   static const String titleWidgetTitle = "Movie detail title";
   static const String titleWidgetSubtitle = "Show the title of the movie";
   static const String examplePath = "/4q2hz2m8hubgvijz8Ez0T2Os2Yv.jpg";
+  static const String defaultTileTitle = "No title for this widget.";
+  static const String defaultTileSubtitle = "No subtitle for this widget.";
+  static const String defaultTileRouteName = "/";
 }
```

Comment:
On this class name you want to say titles right?

Reply:
No, tiles it's right, it's because i used it on the ListTiles but makes sense that it can be titles too, should i change it?

Reply:
Yes, change it please

Reply:
Done